AdAway/src/main/res/values-de/strings.xml

```diff
@@ -68,99 +68,97 @@
-    <string name="apply_symlink_fail">Android konnte keine symbolische Verknüpfung erstellen.\n
-  Das liegt vermutlich an einer Eigenheit von HTC-Geräten namens S-ON!\n
-  \n
-  Eine Lösung ist es, Ihr Gerät im Wiederherstellungsmodus zu starten und dort die Verknüpfung mit »ln -s /data/data/hosts /system/etc/hosts« zu erstellen.\n
-  \n
-  Wenn das nicht funktioniert, suchen Sie bitte im Internet nach S-OFF und Ihrem Gerät.</string>
-    <string name="apply_symlink_fail_title">Symbolische Verknüpfung konnte nicht erstellt werden</string>
-    <string name="apply_help">Bitte die Hilfe für mehr Informationen lesen.</string>
-    <string name="apply_apn_proxy">Auf Ihrem Android-Gerät ist ein Zugangspunktvermittlungsserver (APN-Proxy) festgelegt!\n
-  AdAway wird in Mobilfunknetzen nicht zuverlässig funktionieren. Sie können diesen Vermittlungsserver (Proxy) deaktivieren, indem Sie in den Systemeinstellungen den eingetragenen Wert für den Zugangspunkt löschen (Bei Android 4: Drahtlos und Netzwerke -> Weitere Einstellungen -> Mobile Netzwerke -> Zugangspunkte).</string>
+  Es wird Empfohlen, Android Neu zu Starten, damit die Änderungen wirksam werden. \n
+    \n
+  Möchten Sie Neu Starten? \n
+  (Um Datenverlust zu vermeiden, stellen Sie sicher, dass derzeit keine App, die Speicherkarte nutzt!)</string>
+    <string name="apply_symlink_successful">Symlink wurde erfolgreich erstellt. \n
+  Es wird Empfohlen, Android Neu zu Starten, damit die Änderungen wirksam werden. \n
+    \n
+  Möchtest du Neu Starten? \n
+  (Um Datenverlust zu vermeiden, stellen Sie sicher, dass keine App, die Speicherkarte nutzt!)</string>
+    <string name="apply_symlink_successful_title">Symlink-Erstellung erfolgreich</string>
+    <string name="apply_symlink_fail">Symlink konnte von Android nicht erstellt werden. Dies scheitert meist an einem \"Feature\" namens S-ON auf HTC-Handys! \n
+    \n
+  Eine Lösung besteht darin, das Telefon in den Wiederherstellungsmodus zu starten und den Symlink dort mit \'ln -s / data/ data/hosts/system/etc/hosts\' zu erstellen. \n
+    \n
+  Wenn dies nicht funktioniert, suchen Sie im Internet nach S-OFF und Ihrem Telefon.</string>
+    <string name="apply_symlink_fail_title">Symlink-Erstellung fehlgeschlagen</string>
+    <string name="apply_help">Bitte lesen Sie die Hilfe für weitere Informationen.</string>
+    <string name="apply_apn_proxy">Ein APN-Proxy ist auf Ihrem Android eingestellt! AdAway funktioniert nicht zuverlässig in mobilen Netzwerken wie 3G. Sie können diesen Proxy deaktivieren, indem Sie zum ausgewählten APN wechseln (unter Android 4: Drahtlos und Netzwerke -> Mehr ... -> Mobile Netzwerke -> Zugangspunktnamen) und den Wert im Proxy-Feld entfernen.</string>
     <string name="apply_apn_proxy_title">APN-Proxy eingestellt!</string>
-    <string name="no_connection">Die Internetverbindung funktioniert nicht.</string>
+    <string name="no_connection">Die Verbindung zum Internet funktioniert nicht.</string>
```

Comment:
Why is this better? It is basically the same thing but longer. Also, why not go with `Die Verbindung zum Internet ist unterbrochen.` or something similar?